Jobs/src/Job.IdentityDeletion/Workers/ActualDeletionWorker.cs

```diff
@@ -12,22 +12,29 @@ namespace Backbone.Job.IdentityDeletion.Workers;
 public class ActualDeletionWorker : IHostedService
 {
     private readonly IHostApplicationLifetime _host;
-    private readonly IEnumerable<IIdentityDeleter> _identityDeleters;
     private readonly IMediator _mediator;
     private readonly IPushNotificationSender _pushNotificationSender;
     private readonly ILogger<ActualDeletionWorker> _logger;
+    private readonly IDeletionProcessLogger _deletionProcessLogger;
+    private readonly List<IIdentityDeleter> _remainingIdentityDeleters;
```

Comment:
This one could maybe use a better name, perhaps one that clarifies why you separate IIdentityDeleter objects the way you do.

Reply:
Changes made.